app/components/alchemy/admin/page_select.rb

```diff
@@ -0,0 +1,41 @@
+module Alchemy
+  module Admin
+    class PageSelect < ViewComponent::Base
+      delegate :alchemy, to: :helpers
+
+      def initialize(page = nil, allow_clear: false, placeholder: Alchemy.t(:search_page), query_params: nil)
```

Comment:
maybe add the `url` as optional param as well

```suggestion
      def initialize(page = nil, allow_clear: false, url: alchemy.api_pages_path, placeholder: Alchemy.t(:search_page), query_params: nil)
```